simulations/vip-195/vip-195-testnet/simulation.ts

```diff
@@ -0,0 +1,41 @@
+import { expect } from "chai";
+import { Contract } from "ethers";
+import { ethers } from "hardhat";
+
+import { forking, testVip } from "../../../src/vip-framework";
+import { vip195Testnet } from "../../../vips/vip-195/vip-195-testnet";
+import PRIME_LIQUIDITY_PROVIDER_ABI from "./abis/PrimeLiquidityProvider.json";
+const PRIME = "0xb13Ea8C39594449B2AB5555769580BDB23f5E2Cf";
+
+const PRIME_LIQUIDITY_PROVIDER = "0xce20cACeF98DC03b2e30cD63b7B56B018d171E9c";
+
+forking(34696892, () => {
+  describe("Pre-VIP behavior", () => {
+    let primeLiquidityProvider: Contract;
+
+    before(async () => {
+      primeLiquidityProvider = await ethers.getContractAt(PRIME_LIQUIDITY_PROVIDER_ABI, PRIME_LIQUIDITY_PROVIDER);
+    });
+
+    it("paused", async () => {
+      const paused = await primeLiquidityProvider.paused();
+      // should be true after previous VIPs are executed
+      expect(paused).to.be.equal(false);
+    });
+  });
+
+  testVip("VIP-195 Prime Program", vip195Testnet(), {});
+
+  describe("Post-VIP behavior", async () => {
+    let primeLiquidityProvider: Contract;
+
+    before(async () => {
+      primeLiquidityProvider = await ethers.getContractAt(PRIME_LIQUIDITY_PROVIDER_ABI, PRIME_LIQUIDITY_PROVIDER);
+    });
+
+    it("paused", async () => {
+      const paused = await primeLiquidityProvider.paused();
+      expect(paused).to.be.equal(false);
+    });
+  });
```

Comment:
I would add a check with every step:
1. claim Prime token (from a user with XVS staked more than 90 days ago, we can fake this in the simulation)
2. mine some blocks and invoke `claimInterest`. Check the user receives the expected funds